cypress/support/page_objects/sliderMove.js

```diff
@@ -0,0 +1,36 @@
+export class SliderMove {
```

Comment:
Naming is probably the hardest part of programming, but there are some naming conventions that help a little bit. For example it makes sense to name classes using nouns to tell what it is/represents, and functions/methods with verbs to tell what it does. If this class represents A single move of slider, then its name can be SliderMove and method could be just "move".
But here I would rather say that class represents whole slider, since slider could be considered as a separate page object. So I think we could just name it Slider.

Reply:
I agree with you, nomenclature in programming is really difficult ;-)